### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by sameer-bhardiya**
Updated the getting started on Readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,10 +5,24 @@ This is an [extension for CiviCRM](https://docs.civicrm.org/sysadmin/en/latest/c
 
 ## Getting Started
 
-1. Download the extension
+1. Download the extension in zip file
```
Comment (pokhiii) [inline README.md]:
```suggestion
1. Download the extension: [Download ZIP](https://github.com/pokhiii/sourcetracker/archive/refs/heads/main.zip)
```

File: README.md [hunk 2]
```diff
@@ -5,10 +5,24 @@ This is an [extension for CiviCRM](https://docs.civicrm.org/sysadmin/en/latest/c
...
+    - Extract the zip file and named as sourcetracker
```
Comment (pokhiii) [inline README.md]: This is not sub-point of the first point
```suggestion
1. Extract the zip file and rename it from `sourcetracker-main` to simply `sourcetracker`
```

File: README.md [hunk 3]
```diff
@@ -5,10 +5,24 @@ This is an [extension for CiviCRM](https://docs.civicrm.org/sysadmin/en/latest/c
...
+    - Changed the directory of file to civicrm\wp-content\uploads\civicrm\ext
```
Comment (pokhiii) [inline README.md]:
```suggestion
    - Move the extension (unzipped) to your CiviCRM extension directory. Eg. in WordPress it is will be `path/to/wordpress/wp-content/uploads/civicrm/ext`
```

File: README.md [hunk 4]
```diff
@@ -5,10 +5,24 @@ This is an [extension for CiviCRM](https://docs.civicrm.org/sysadmin/en/latest/c
...
 2. Install the extension
-3. Create a custom field named `Source` (the name should be exactly same & case-sensitive)
-4. Include this field in any Profile (as type hidden)
+    - Navigate to the **Administer** > **System Setting** > **Extension**
```
Comment (pokhiii) [inline README.md:13]: Add one more point above it – `Login to your CMS and navigate to the CiviCRM dashboard`

File: README.md [hunk 5]
```diff
@@ -5,10 +5,24 @@ This is an [extension for CiviCRM](https://docs.civicrm.org/sysadmin/en/latest/c
...
+    - Find **Source Tracker** in extension and install it (Refresh the page if you can't see the source tracker extension)
+3. Create a custom field named `Source`
+    - Navigate To the **Administer** > **Customize Data and Screen** > **Custom Field**
+    - Click on **Add Set of Custom Fields**
+    - Create custom field named `Source` (the name should be exactly same & case-sensitive) and mark **hidden** on Field Input Type *
```
Comment (pokhiii) [inline README.md]:
```suggestion
    - Create a custom field named `Source` (the name should be the same & case-sensitive) and mark **hidden** on Field Input Type
```

File: README.md [hunk 6]
```diff
@@ -5,10 +5,24 @@ This is an [extension for CiviCRM](https://docs.civicrm.org/sysadmin/en/latest/c
...
-4. Include this field in any Profile (as type hidden)
+    - Navigate to the **Administer** > **System Setting** > **Extension**
+    - Find **Source Tracker** in extension and install it (Refresh the page if you can't see the source tracker extension)
+3. Create a custom field named `Source`
+    - Navigate To the **Administer** > **Customize Data and Screen** > **Custom Field**
+    - Click on **Add Set of Custom Fields**
+    - Create custom field named `Source` (the name should be exactly same & case-sensitive) and mark **hidden** on Field Input Type *	
+	
+4. Use this field in any Profile
+   - Include this field in any Profile (as type hidden)
```
Comment (pokhiii) [inline README.md]:
```suggestion
   - Include this field in any Profile
```

File: README.md [hunk 7]
```diff
@@ -5,10 +5,24 @@ This is an [extension for CiviCRM](https://docs.civicrm.org/sysadmin/en/latest/c
...
+   - Use the profile in the registration form
```
Comment (pokhiii) [inline README.md]:
```suggestion
   - Use the Profile for registrations. Eg. in WordPress it is equivalent to using the shortcode on a WordPress page.
```

File: README.md [hunk 8]
```diff
@@ -5,10 +5,24 @@ This is an [extension for CiviCRM](https://docs.civicrm.org/sysadmin/en/latest/c
...
+   - Add the UTM parameter at the end of the registration form page URL. ( like  `?utm_source=whatsapp` )
```
Comment (pokhiii) [inline README.md]:
```suggestion
   - Add the UTM parameter at the end of the registration form page URL. Eg. `?utm_source=whatsapp`
```

File: README.md [hunk 9]
```diff
@@ -5,10 +5,24 @@ This is an [extension for CiviCRM](https://docs.civicrm.org/sysadmin/en/latest/c
...
+   - Submit the registration form
+   - Navigate to the **Search** > **Find contacts**
+   - Click on the contract you registered; you will see the source field at the bottom of the detail form
```
Comment (pokhiii) [inline README.md]:
```suggestion
   - Click on the contact you registered; you will see the Source field filled with the value in the `utm_source`
```

**Review by pokhiii — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (sameer-bhardiya) [reply README.md:13]: ok sure

**Review by pokhiii — APPROVED** [review]

**Pull request merged**